apps/root/src/pages/strategy-guardian-detail/vault-data/components/data-guardian/index.tsx

```diff
@@ -0,0 +1,164 @@
+import { DisplayStrategy } from 'common-types';
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import styled from 'styled-components';
+import {
+  Typography,
+  GlobalIcon,
+  colors,
+  ContainerBox,
+  Skeleton,
+  ArrowDropDownIcon,
+  ArrowDropUpIcon,
+  Collapse,
+  ForegroundPaper,
+  Link,
+  TwitterIcon,
+  DiscordIcon,
+} from 'ui-library';
+import { SPACING } from 'ui-library/src/theme/constants';
```

Comment:
This function should not ve available like this, we should use it through the theme

Reply:
We actually use it quite a bit. Its good when we dont want to introduce a hook to a component since the spacing is going to be constant
<img width="393" alt="Screenshot 2024-08-23 at 2 00 52 PM" src="https://github.com/user-attachments/assets/6bad25b1-c40c-4f61-adee-7af8e6c19e28">